apps/events-helsinki/src/common-events/components/shareLinks/ShareLinks.tsx

```diff
@@ -13,7 +14,8 @@ export interface ShareLinksProps {
 }
 
 const ShareLinks: React.FC<ShareLinksProps> = ({ title }) => {
```

Comment:
A bit out of scope of this PR maybe, but this ShareLinks could be moved to common components-package.